applications/nrf5340_audio/src/bluetooth/bt_rendering_and_capture/volume/bt_vol_ctlr.c

```diff
@@ -121,33 +249,43 @@ static void vcs_flags_ctlr_cb_handler(struct bt_vcp_vol_ctlr *vcs, int err, uint
 	}
 }
 
-/**
- * @brief	Callback handler for the finished VCS discovery.
- *
- * @note	This callback handler will be triggered when the VCS discovery has finished.
- */
```

Comment:
Why was the documentation removed?

Reply:
These are static functions and their documentation should reside where this callback is defined in the host.